lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width-open: 250px;
+$nx-sidebar-width-closed: 72px;
+$nx-sidebar-logo-height: 34px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  box-sizing: border-box;
+  color: $nx-sidebar-text-color;
+  max-width: $nx-sidebar-max-width;
+  padding: $nx-spacing-xl $nx-spacing-l $nx-spacing-l $nx-spacing-l;
+  width: $nx-sidebar-width-open;
+
+  .nx-text-link {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-page-sidebar-operable__header {
+  display: flex;
+  margin-bottom: $nx-spacing-xl;
+}
+
+.nx-page-sidebar-operable__toggle {
+  border-color: transparent;
+  border-radius: 50%;
+  color: $nx-sidebar-text-color;
+  flex: none;
+  height: 34px;
+  margin: 0;
+  width: 34px;
+
+  &:focus {
+    border-color: $nx-focus-border-color;
+  }
+
+  &:active {
+    background-color: $nx-sidebar-active-color;
+  }
+
+  &:hover {
+    background-color: $nx-sidebar-hover-color;
+  }
+}
+
+.nx-page-sidebar-operable__product-info {
+  flex: 1 1 auto;
+  height: $nx-sidebar-logo-height;
```

Comment:
Somewhere along the way, the `height` on the `<img>` itself got lost.

If I'm not mistaken, this `height` here is to prevent the `<a>` from having some extra space underneath the `<img>`, right?  There are two better ways to fix that:
* give the `<img>` a `vertical-align` other than `baseline` (for instance `top` or `bottom`)
* Make the `<a>` a flex container.

Reply:
> * Make the `<a>` a flex container.

I had that and actually thought this was better. 🤦🏽

Reply:
> I had that and actually thought this was better. 🤦🏽

So, the reason it's not is because it requires another specific number that has to be kept in sync.

Reply:
> Somewhere along the way, the `height` on the `<img>` itself got lost.
> 
> If I'm not mistaken, this `height` here is to prevent the `<a>` from having some extra space underneath the `<img>`, right? There are two better ways to fix that:
> 
> * give the `<img>` a `vertical-align` other than `baseline` (for instance `top` or `bottom`)
> * Make the `<a>` a flex container.

Went with `flex` in 92642288444c4e7fdca8f5439d4d3be6e0b60ded